mongomock/collection.py

```diff
@@ -1173,11 +1167,11 @@ def group(self, key, condition, initial, reduce, finalize=None):
         for k in key:
             doc_list = sorted(doc_list, key=lambda x: _resolve_key(k, x))
         for k in key:
-            if not isinstance(k, basestring):
+            if not isinstance(k, helpers.basestring):
                 raise TypeError(
                     "Keys must be a list of key names, "
                     "each an instance of %s" %
-                    (basestring.__name__,))
+                    (helpers.basestring.__name__,))
```

Comment:
It seems pretty useless in this case to create an empty tuple here as the old-style format string also take a simple variable after the `%`